sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/predicates.scala

```diff
@@ -442,8 +442,6 @@ case class Or(left: Expression, right: Expression) extends BinaryOperator with P
 
   override def inputType: AbstractDataType = BooleanType
 
-  override def symbol: String = "||"
```

Comment:
I think this won't be compiled?

```
class Or needs to be abstract, since method symbol in class BinaryOperator of type => String is not defined
```

Reply:
I am sorry for this error, I will try to avoid it in the future.